Rubberduck.Resources/RubberduckUI.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace Rubberduck.Resources {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]
```

Comment:
It's weird that this seem to be the only place where a change was made. It was most likely done by Visual Studio in the background. Unfortunately I don't know enough whether the change in version will cause an issue. Because that has nothing to do with the translation, I'm inclined to want to avoid changing this.

Reply:
17 is VS2022, isn't it? I think we can leave it now.

Reply:
Yes but why not the other files? We have several `*.Designer.cs` that still has 16 and only this one was updated?

Reply:
Actually while migrating resources to RD3 I noticed the .designer files weren't consistently present; I think we only actually need the English ones, to generate the C# code so the keys can be referenced in code.